docs/API/Elements.md

```diff
@@ -93,6 +143,14 @@ element.schema.setBool('fizz', true);
 element.schema.getBool('fizz');    // true
 ```
 
+Here is an example using scheme to get the object name and rename. (Will not last outside playmode and does not change in the inspector.)
+```js
+  var name = this.schema.getString('name'); //get current object name
+  log.info("my name is " + name); //output current name 
+  this.schema.setString('name','Bob'); //change name
+  log.info("my name is " + this.schema.getString('name')); //output newly changed name
+```
+
 <br>
 
 If an element doesn't already have a value, it returns a value up the graph. In this circumstance, the property of b is bound to the property of a.
```

Comment:
I wonder if it would be helpful to have "b" and "a" displayed as `b` and `a`

Reply:
Considering it is placeholder code I don't know. If it was actually like a keyword or useable code I would 100% be for it, but I don't know if it could be confusing if someone just thinks they can copy that code below and run it without modifying it. I can make it italic or bold though as that is what I'm doing for other named variables.